Source/WebCore/PAL/pal/HysteresisActivity.h

```diff
@@ -61,17 +61,15 @@ class HysteresisActivity {
     {
         if (!m_active)
             return;
-        m_active = false;
 
+        m_active = false;
         m_timer.startOneShot(m_hysteresisSeconds);
     }
 
     void impulse()
     {
-        if (!m_active) {
-            start();
-            stop();
-        }
+        if (!m_active)
+            m_timer.startOneShot(m_hysteresisSeconds);
```

Comment:
If `!m_timer.isActive()`, it seems we fail to call `m_callback(HysteresisState::Started)` now no?